src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapSiriusToBSC.vue

```diff
@@ -206,9 +206,9 @@ export default {
       selectedAccountName.value = name;
       selectedAccountAddress.value = address;
 
-      if(bscGasStrategy.value === ""){
-        changeGasStrategy("standard");
-      }
+      // if(bscGasStrategy.value === ""){
+      changeGasStrategy("standard");
+      // }
```

Comment:
by comment it out, it will change back to standard price when user change account

Reply:
It's alright is it when we set the fee to default whenever the account is changed 🙂